packages/ui/src/components/Form/styles.css.ts

```diff
@@ -41,8 +41,10 @@ export const inputVariants = recipe({
 				width: 0,
 				padding: 0,
 				border: 'none',
-				'&:focus, &:active, &:placeholder-shown:hover': {
```

Comment:
This was ending up in the final CSS output apparently (screenshot from prod):

![Screenshot 2023-04-17 at 5 42 56 PM](https://user-images.githubusercontent.com/6934200/232640144-c0b5f109-6e79-4780-a9d5-699501b168ab.png)


Not valid syntax outside of browsers with draft CSS nesting support, though support is actually surprisingly good for a draft: https://caniuse.com/css-nesting